<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="keyEvent">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>391</height>
     </rect>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>291</width>
       <height>391</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="image">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>511</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>801</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QPushButton" name="gripperLeftButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="hallerUpButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>30</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="hallerLeftButton">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>90</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="hallerDownButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>90</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="hallerRightButton">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>90</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="gripperRightButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="gripperDownButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="gripperUpButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="gripperLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Gipper</string>
     </property>
    </widget>
    <widget class="QLabel" name="hallerLabel">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>10</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Haller</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
